shared/reducers/config.js

```diff
@@ -100,32 +117,32 @@ export default function (state: ConfigState = initialState, action: Action): Con
         const status = action.payload.status
         return {
           ...state,
-          loggedIn: status.loggedIn,
           status,
-          uid: status.user && status.user.uid,
-          username: status.user && status.user.username,
         }
       }
       return state
 
-    case Constants.bootstrapAttemptFailed: {
+    case Constants.bootstrapLoaded:
+      const {bootstrapStatus} = action.payload
       return {
         ...state,
-        bootstrapTriesRemaining: state.bootstrapTriesRemaining - 1,
+        ...bootstrapStatus,
+        following: arrayToObjectSet(bootstrapStatus.following),
```

Comment:
The service gives us to these as arrays, but we expect them to be objects w/ each key having value `true`. Would be nice to convert these to `Set`s in the future.